docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -12,7 +12,7 @@ In this guide you will:
 
 ## Prerequisites
 
-- Teleport version `7.0` or newer.
+- Teleport version `(=teleport.version=)` or newer.
```

Comment:
If we always put the latest version here, there will be no "or newer".

Shouldn't this list the version in which MongoDB support was added.

Reply:
I would be concerned about the following scenario:

- Teleport customer (running 7.3 for example) Googles "How to connect a MongoDB database to Teleport"
- Google takes them to the relevant docs page, which by default loads the latest version.
- The page says "Prerequisites: Teleport 9.0.4 or newer"
- The user stops reading and doesn't try to use this feature because they got the impression that their version of Teleport is too old.

Reply:
The pattern I usually try to follow is:
- In the prerequisites, hardcode the version the feature was introduced in. So here I'd keep "Teleport version 7.0 or newer".
- In the download instructions (e.g. "install teleport x.y.z") use `(=teleport.version=)` variable which always point to the current.

Reply:
@r0mant In this case how should we handle RDS for example: https://github.com/gravitational/teleport/blob/master/docs/pages/database-access/guides/rds.mdx
Currently we're using `teleport.version` in both places. If we want to keep the lowest required version then we need to say
* 7.0 If you use RDS
* 8.0 if you use Aurora
* 9.0 if you use MariaDB
Am I correct?

Reply:
I'd probably say something like 

```
* Teleport version 7.0 or newer. MariaDB RDS support requires Teleport 9.0 or newer.
```

Reply:
Added